src/network/socks5.py

```diff
@@ -121,14 +140,17 @@ def state_proxy_port(self):
         return True
 
     def proxy_sock_name(self):
+        """"""
         return socket.inet_ntoa(self.__proxysockname[0])
 
 
 class Socks5Connection(Socks5):
+    """"""
```

Comment:
Init

Reply:
Went for """Child socks5 class used for making outbound connections.""", assuming you were thinking about commenting the `__init__` method with your comment